### ISSUE TITLE
{1.11}Spectre key stopped working

### ISSUE THREAD

**Issue by IronworksProductions** [lumien231/Random-Things#299]
Hey,

So I ran into an issue where after a few days in the world the spectre key stops working and i am no longer able to access the spectre dimension in that world.
log:https://hastebin.com/qotudujaqu.sql

**Comment by Dusty-Sprinkles** [comment]
This is also a problem in 1.12.2, though I can't pull any logs since I don't own the server, but when I ran a brand new singleplayer world with the same configuration, it worked fine.

**Comment by IronworksProductions** [comment]
Yes i can confirm that it is also an issue on 1.12.2 I really hope this mod is not abandoned it was so useful but this has not been touched in quite a while

**Comment by lumien231** [comment]
Abandoned? The last commit was 2 days ago ;).

There's nothing in your log relating to the spectre dimension so it's a little hard to fix this issue.

**Comment by IronworksProductions** [comment]
Yeah I can understand that. I could not find any evidence in the logs but I use the key then it cancels and does not tp. It does not always do it though a brand new world will sometimes take you to the spectre dimension. Unless it is your second single player world then it will not tp.

**Comment by brisingraerowing** [comment]
This happens sometimes to me. It's kinda random. I think it's a compat issue with at least one other mod.

And now I'm going back to debugging why I get kicked to the MP selection screen when reloading my SP world. It only happens after I go to the spectre dimension, so I think it may have something to do with that, but I'm not sure. Probably a mod compat issue there as well.
